### DISCUSSION TITLE
Playtime is not defined

### DISCUSSION THREAD

**Discussion by meitsme1900** [isid0re/SoloLeveling#155] [General]
![image](https://user-images.githubusercontent.com/79985007/109815815-bc0e5200-7c38-11eb-851a-db6e6a713a48.png)

Hello im new to d2 and botting it, trying this for the first time thanks for making this :)
do i need to edit something here?

![image](https://user-images.githubusercontent.com/79985007/109816118-0db6dc80-7c39-11eb-9bed-eb905554b2dc.png)

**Comment by isid0re** [comment]
Sounds like a bad install. Post a screen shot of your d2bs/kolbot/libs/ folder

**Reply by meitsme1900** [reply]
https://user-images.githubusercontent.com/79985007/109834732-2def9700-7c4b-11eb-91db-b86bf464b97e.jpg

will try another install meanwhile ty.
Edit: worked after new install guess ive messed something up ty.
